### ISSUE TITLE
Event on calendar is displaying the date 

### ISSUE THREAD

**Issue by Seqi** [Seqi/raifu#201]
**Current Behavior**

Location: App -> Events Tab 

Using the monthly view, add an event, so that an event is displayed on the calendar. The event is displayed in the format:

```
EventName
EventLocation @ Date Time
```

As the calendar already displays the date, the `Date` part is redundant, and takes up much needed space.

**Desired Behavior**

The events in the monthly calendar view should not display Date, only Time:

```
EventName
EventLocation @ Time
```

**Notes:**
The current format is retrieved via subtitle (`entity.model.js`). We don't want to change this. Therefore we can just ditch the `toSubtitle()` call and just do it all in jsx format.

**Comment by Seqi** [comment]
The culprit lies in `CalendarEvent.js`

**Comment by Seqi** [comment]
Fixed in [5e041a72675b0884faeba47a6247aa877503c5b5](https://github.com/Seqi/raifu/commit/5e041a72675b0884faeba47a6247aa877503c5b5)
